### ISSUE TITLE
Pet carrier wall glitch

### ISSUE THREAD

**Issue by XxSWAG-MASTER-OVERLORDxX** [space-wizards/space-station-14#28224]
## Description
A bug with a pet carrier that lets you put people through windoors, ive seen it used a few times now and while i cant get it to work myself and it seems to have a low likelihood of working for those that try it. 

It seems to work by pressing it up past the windoor somehow

**Reproduction**
Sadly i am unable to replace it but if anyone has more info on how this works it would be great

I understand if this isnt enough info to go on for an issue post and if you decide to close it based on that, its fine.

**Comment by 0x6273** [comment]
Same as https://github.com/space-wizards/space-station-14/issues/25457 ? It's supposed to have been fixed.

**Comment by themias** [comment]
Yes, this looks like the same issue fixed by #27137. I can't replicate the pet carrier windoor trick locally. If someone can replicate this, please leave a comment and we can reopen the issue.
